src/wix/WixToolset.Core/ExtensibilityServices/WixBranding.cs

```diff
@@ -101,6 +102,20 @@ public string ReplacePlaceholders(string original, Assembly assembly = null)
                 original = original.Replace("[AssemblyTitle]", title.Title);
             }
 
+            try
+            {
+                var assemblyVersion = 
+                    ThisAssembly.Git.BaseVersion.Major + "." +
+                    ThisAssembly.Git.BaseVersion.Minor + "." +
+                    ThisAssembly.Git.BaseVersion.Patch +
+                    ThisAssembly.Git.SemVer.DashLabel + "-" +
+                    ThisAssembly.Git.Branch + "+" +
+                    ThisAssembly.Git.Commit;
+
+                original = original.Replace("[AssemblyVersion]", assemblyVersion);
+            }
+            catch { }
```

Comment:
I probably should have expounded upon this comment a bit more:

> Better to update WixBranding to add a new ProductVersion replacement and use that. Then this won't be a special casing using ThisAssembly.

We should avoid doing this string concatenation that is already done in the build process to create the versions in the files themselves. To ensure the values displayed here match the values in the files, let's use the file version information in the files themselves.

Reply:
To do so, add lines near 64 and 72 to do the `[ProductVersion]` replacement much the same as the `[FileVersion]` replacement is done today.